test/src/test/java/org/corfudb/runtime/checkpoint/CheckpointTrimTest.java

```diff
@@ -22,6 +22,8 @@
  * Created by mwei on 5/25/17.
  */
 public class CheckpointTrimTest extends AbstractViewTest {
+    final static long NUM_ENTRIES = 5000;
+    final static long Round3 = 3;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4685528)